docs/drivers/exoscale.md

```diff
@@ -32,6 +32,8 @@ Options:
 
 If a custom security group is provided, you need to ensure that you allow TCP ports 22 and 2376 in an ingress rule. Moreover, if you want to use Swarm, also add TCP port 3376.
 
+There is a limit to the number of anti-affinity groups a machine can belong to.  You can work around it by specifying a second anti-affinity group using `--exoscale-affinity-group=docker-machineX`
```

Comment:
It's the reverse. There is a limit to the number of machines an anti-affinity group can contain.